modules/tile-converter/src/i3s-converter/i3s-converter.js

```diff
@@ -88,7 +88,7 @@ export default class I3SConverter {
 
     const fetchOptions = await this._fetchPreloadOptions();
     const sourceTilesetJson = await load(inputUrl, CesiumIonLoader, {
-      fetch: fetchOptions
+      fetch: {fetch: {headers: fetchOptions.headers}, throws: true}
```

Comment:
This is too confusing. Let's land #1549 first.

Reply:
I updated PR after your changes [here](https://github.com/visgl/loaders.gl/pull/1549)